.github/workflows/check-license.yml

```diff
@@ -14,6 +14,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: refs/pull/${{ github.event.number }}/merge
```

Comment:
fore reviewer, refer to https://stackoverflow.com/a/63595981 for the meaning of `/refs/pull/<num>/merge`